### PR TITLE
Include kgraft-patchname in qadb-submission comment

### PR DESCRIPTION
**Pull request by fschilling**
For convenience: Find the currently applied and active kgraft-patch and add it as a comment to the qadb-submission.

### CODE REVIEW

File: tests/qam-kgraft/reboot_restore.pm [hunk 1]
```diff
@@ -49,7 +49,7 @@ sub run() {
     upload_logs("/tmp/var_log_qa_ctcs2.tar.gz");
 
     script_run("ssh-keygen -R qadb2.suse.de");
-    assert_script_run(qq{/usr/share/qa/tools/remote_qa_db_report.pl -L -b -T openqa -c "`uname -r -v`" -t patch:"$rrid" &> /tmp/submission.log }, 1800);
+    assert_script_run(qq{/usr/share/qa/tools/remote_qa_db_report.pl -L -b -T openqa -c "`uname -r -v` `kgr -v patches | grep -B2 RPM | head -n1`" -t patch:"$rrid" &> /tmp/submission.log }, 1800);
```
Comment (okurz) [inline tests/qam-kgraft/reboot_restore.pm]: line is getting too long. That is also the reason for the check failure. Please make this a bit more readable, e.g. set parts of the string as variables and include them here and/or break lines.

**Comment by fschilling** [comment]
As requested I changed it to be multi-line with one parameter per line.

**Comment by okurz** [comment]
checks failed, some minor style issue remaining.

**Comment by fschilling** [comment]
fixed the style issues.

**Comment by okurz** [comment]
looks weird but nevermind for now :-)

**Pull request merged**